docs/guide-authentication.md

```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
 Selecting e.g. Google will redirect the browser to their website so the user can authenticate there and return to MOLGENIS once authentication has successfully completed.
 The permissions you have once authenticated are the default user permissions set by an administrator.   
 
-### Configuring single sign-on   
+## Configuring single sign-on   
 As administrator single sign-on configuration consists of the following one-time steps:
-1. Configuring one or more OpenID Connect client
-2. Modify authentication settings
-
-#### Configuring OpenID Connect clients
+1. Register MOLGENIS with the ID providers
+2. Configure one or more OpenID Connect clients, one for each identity provider
+2. Modify the authentication settings, activating clients for the identity providers.
+
+### Registering MOLGENIS with an ID provider
+Your MOLGENIS server needs to be registered with the identity provider as a client.
+The identity provider typically has a web interface where you can do this.
+
+*  A `client ID` and a `client secret` are issued by the identity provider and used
+by MOLGENIS to authenticate with the provider.
+*  The provider will typically also ask you how it can validate the
+`redirect URI`.
+The redirect URI is an endpoint on the MOLGENIS server: 
+`https://<serverURL>/login/oauth2/code/<registrationID>`
+The registration ID is an ID that you assign to the provider when you register the
+OIDC client.
+*  Since the user will be authenticating with the ID provider in the browser,
+the provider may allow you to add some branding to the login page,
+such as a logo or styling.
+
+### Registering an ID provider with MOLGENIS
+MOLGENIS uses an OpenID Connect client to communicate with the ID provider.
 OpenID Connect clients can be configured by adding entities to the 'OIDC client' entity type (e.g. using the data explorer, importer or REST API).
```

Comment:
![SOPoidc](https://user-images.githubusercontent.com/40240913/70709896-4ab64c80-1cde-11ea-8866-f8e703800527.png)
Example OIDC client, ID and secret are crossed out.

Reply:
I thought about this a bit more, and nowadays the best way to find an up to date version of the endpoints is the configuration endpoint. Updated the table below with a couple of example endpoints.

Reply:
Added the MITRE / perun config endpoint to the table